README.md

```diff
@@ -1,6 +1,6 @@
 # bitbucket-pipelines dotnet-build
 
-[![logo](./logo.jpg)](https://inforit.nl)
+[![logo](./logo.jpg)](https://frontliners.nl)
```

Comment:
Kindly change the logo to be frontliners as well. 
The logo can be found in the brand guide 😄